core/base-service/base.js

```diff
@@ -108,11 +108,14 @@ class BaseService {
    *
    * See also the config schema in `./server.js` and `doc/server-secrets.md`.
    *
-   * To use the configured auth in the handler or fetch method, pass the
-   * credentials to the request. For example:
-   * - `{ options: { auth: this.authHelper.basicAuth } }`
-   * - `{ options: { headers: this.authHelper.bearerAuthHeader } }`
-   * - `{ options: { searchParams: { token: this.authHelper._pass } } }`
```

Comment:
Were these inaccurate already/currently, independent of the got migration? Interesting if so, seems like we've discovered a couple inaccuracies along the way as part of this migration.

Reply:
Yes. I updated this from
`{ options: { qs: { token: this.authHelper._pass } } }`
to
`{ options: { searchParams: { token: this.authHelper._pass } } }`
in 
49476e3

Then I was going to update the `auth` key and realised that it just didn't really make sense.